drizzlepac/wcs_functions.py

```diff
@@ -1069,15 +1069,34 @@ def make_mosaic_wcs(filenames, rot=None, scale=None):
     mosaic_wcs : HSTWCS object
         the merged composite WCS
     """
+    
+    # makes str into list of str
     if not isinstance(filenames, list):
         filenames = [filenames]
 
+    # checks if drz/drc or flc/flt identifies type with extension var
+    extension='None'
+    if np.all(['flc' in x or 'flt' in x for x in filenames]):
```

Comment:
```suggestion
    if all(['flc' in x or 'flt' in x for x in filenames]):
```
no need to convert this to `numpy` arrays.